src/Main.elm

```diff
@@ -346,23 +372,12 @@ renderGameArea model =
         renderCardList model.cards
 
 
-renderHelp : Model -> Html Msg
-renderHelp model =
-    if model.helpClosed then
-        button [ class "help", onClick PressedHelp, ariaExpanded "false" ] [ h2 [] [ text "How to play +" ] ]
-
-    else
-        div []
-            [ button [ class "help", onClick PressedHelp, ariaExpanded "true" ] [ h2 [] [ text "How to play -" ] ]
-            , if model.helpClosed then
-                text ""
-
-              else
-                div [ class "help-text" ]
-                    [ p [] [ text model.cardSetMeta.help ]
-                    , p [] [ text "Why not challenge yourself to guess it right first time?" ]
-                    ]
-            ]
+renderArtistSpeech : String -> Html Msg
+renderArtistSpeech speech =
+    div [ class "artist-speech container" ]
+        [ img [ class "artist", src "card-images/barbara.svg" ] []
```

Comment:
Missing alt, should probably convey that the text is from a person.

Reply:
Good point. I'll add.